dbconnection/DatabaseLink.java

```diff
@@ -0,0 +1,42 @@
+package com.company.dbconnection;
+
+import com.company.Constants;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class DatabaseLink {
+
+    public void DatabaseConnection(String textFilePath, String searchKeyword, String searchResult, String errorMessage, int count) {
```

Comment:
change the method name as per java naming conventions

Reply:
Made the change.